funcx_sdk/funcx/sdk/client.py

```diff
@@ -112,6 +114,12 @@ def __init__(
         Keyword arguments are the same as for BaseClient.
 
         """
+        # resolve URLs if not set
+        if funcx_service_address is None:
+            funcx_service_address = get_web_service_url(environment)
+        if results_ws_uri is None:
+            results_ws_uri = get_web_socket_url(environment)
```

Comment:
@sirosen Could we get a couple basic tests of this by checking the values of `funcx_service_address` and `results_ws_uri` on the client after initialization similar to what I was doing here: https://github.com/funcx-faas/funcX/pull/655/files#diff-3f34ddd517aac9398aec65e15d62d6c3a02ed8f38a6d4bdc8f1168bcddee68db

Reply:
Yeah, that's a very reasonable addition to the tests. I'll add that as well, and may crib from the tests you wrote.

Reply:
You can see how I handled this in a highly parametrized test here: 738e4aa (I could have written several distinct tests; this was just the fastest way for me to write this matrix of conditions.)
I'm going to mark this resolved.

I'm curious at some point to know how you were running your tests, since `FuncXClient.__init__` involves a _lot_ of externalities -- network callouts and local filesystem access -- which I had to mock over.